script/deploy/ValidProposal.s.sol

```diff
@@ -21,6 +21,7 @@ contract DeployProposal is Script, zip {
     bool public doAfterdeploy;
     bool public doValidate;
     bool public doTeardown;
+    string constant ADDRESSES_PATH = "proposals/Addresses/mainnet.json";
```

Comment:
Shouldn't this path be dynamic?

Reply:
I assumed that we are using script only for mainnet. I can make it dynamic as well.

Reply:
No, it should be available for any network.

Reply:
fixed this here [4a96828](https://github.com/ZTX-Foundation/tuxedo/pull/203/commits/4a968281b6caa97c4fd60b3c4237a773ab09bc33)